packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -28,6 +28,7 @@ export enum ProfilesConvertStatus {
 
 export class ProfilesUtils {
     public static PROFILE_SECURITY: string | boolean = Constants.ZOWE_CLI_SCM;
+    private static noConfigDialogShown: boolean = false;
```

Comment:
Should we rename this variable so it doesn't start with the word `no`? Please feel free to ignore, I'm definitely being picky but this was a bit hard to read because it seemed like a double negative 😋 

```suggestion
    private static createConfigDialogShown: boolean = false;
```